articles/custom-domains/additional-configuration.md

```diff
@@ -129,15 +149,22 @@ If you want to use SAML applications with your custom domain, you must update yo
 
 If you have an IdP-initiated authentication flow, you will need to update the URL used to invoke the IdP-initiated authentication flow to reflect the custom domain. Instead of `https://<TENANT>.auth0.com/saml/<CLIENTID>` you should use `https://<CNAME>/saml/<CLIENTID>`.
 
-## APIs
+If you use the Auth0 APIs, such as the Management API, the API identifier will use your default tenant domain name (such as `https://${account.namespace}/userinfo` and `https://${account.namespace}/api/v2/`)
 
-If you use Auth0 with a custom domain to issue Access Tokens for your APIs, then you must validate the JWT issuer(s) against your custom domains. For example, if you use the [express-jwt](https://github.com/auth0/express-jwt) middleware, you must do the following change.
+## Configure your WS-Fed Clients
 
-```js
-app.use(jwt({ 
-  issuer: 'https://YOUR-CUSTOM-DOMAIN',
-  //code omitted for brevity
-}));
-```
+If you want to use your WS-Fed applications with your custom domain with Auth0 as the IDP, you must update your Service Provider with new Identity Provider metadata from Auth0 (You can obtain the metadata reflecting the custom domain from: `https://<CUSTOM DOMAIN>/wsfed/FederationMetadata/2007-06/FederationMetadata.xml`).
 
-If you use the Auth0 APIs, such as the Management API, the API identifier will use your default tenant domain name (such as `https://${account.namespace}/userinfo` and `https://${account.namespace}/api/v2/`)
+## Configure Azure AD connections
+
+If you want to use Azure AD connections with your custom domain, you must update the Allowed Reply URL in your Azure AD settings. In your Azure Active Directory, go to **Apps registrations** and select your app. Then click **Settings -> Reply URLs** and add a URL in the format `https://<CUSTOM DOMAIN>/login/callback` that includes your custom domain (such as `https://login.northwind.com/login/callback`).
+
+## Configure ADFS connections
+
+The process is the same as [setting up the ADFS connection normally](/connections/enterprise/adfs) except that your callback URL needs to be changed from this format `https://<TENANT>.auth0.com/login/callback` to this one use `https://<YOUR-CUSTOM-DOMAIN>/login/callback`.
+
+## Configure AD/LAP connections
+
+In order to use AD/LAP connections with Kerberos support and custom domains, you will need to update the ticket endpoint to work with the custom domain. [As mentioned in the [Auth0 AD/LAP connector documentation](/connector/modify#point-an-ad-ldap-connector-to-a-new-connection), the `config.json` file needs to be modified, with the `PROVISIONING_TICKET` value changed from this format `https://<TENANT>.auth0.com/p/ad/jUG0dN0R/info` to `https://<CUSTOM DOMAIN>/p/ad/jUG0dN0R/info`.
```

Comment:
typo: AD/LAP

Reply:
fix: `[As mentioned in the [Auth0 AD/LAP connector documentation](/connector/modify#point-an-ad-ldap-connector-to-a-new-connection)`

Reply:
This talks only about Kerberos support but on the other doc me also mention without Kerberos support. Is the process different or the same?

Reply:
add this info also at this doc https://auth0.com/docs/connector/modify#point-an-ad-ldap-connector-to-a-new-connection